src/main/java/org/mskcc/cmo/metadb/service/ValidRequestChecker.java

```diff
@@ -0,0 +1,10 @@
+package org.mskcc.cmo.metadb.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import java.io.IOException;
+
+public interface ValidRequestChecker {
+    public String checkIfValidRequest(String requestJson)
```

Comment:
I'm thinking we should rename this method since the current name corresponds more to a method that would be returning a boolean but instead here we are returning a JSON that's potentially a filtered version of the JSON we input. 

So maybe something like `getValidRequestJson` or `getFilteredValidRequestJson`